configuration/items.md

```diff
@@ -242,40 +258,80 @@ FIRST_QUARTER=🌓 First Quarter
 ```
 
 ## Icons
-OpenHAB provides you a set of [basic icons]({{base}}/addons/iconsets/classic/readme.html) by default.
+OpenHAB provides a set of [basic icons]({{base}}/addons/iconsets/classic/readme.html) by default.
 However if you wish to use custom icons you need to place them inside the `conf/icons/classic/` folder.
 These icons will be used in all of the openHAB frontends.
-The images must be in .png or .svg format, and have a name with only small letters and a hyphen or underscore (if required).
-The PaperUI interface (or via the classicui.cfg or basicui.cfg files) allows you to define whether you use Vector (.svg) or Bitmap (.png) icon files.
 
-As an example, to use a custom icon called `heatpump.svg` the correct syntax is `<heatpump>`.
+The images must be in `.png` or `.svg` format, consist of only lowercase letters, hyphens `-` or underscores `_` (if required).
```

Comment:
This is a very important topic. We should make it as clear as possible.

Custom Icons must abide to the following file name restrictions to be usable in openHAB
* `png` or `svg` format (see below)
* Only lowercase letters and underscores `_`
* Hyphens `-` are used in dynamic icons (see below), hyphens are not allowed as part of the filename
* *No* uppercase letters, numbers or other special characters.
* Examples:
  * Allowed: `switch.svg`, `power_meter.png`
  * Not allowed: `PC_Monitor.svg`, `light_1st_floor.png`, `power-meter.png`, `tür.svg`